platform/android/DEVELOPING.md

```diff
@@ -58,6 +58,14 @@ final draft of the C++14 standard. More information in [DEVELOPING.md](../../DEV
 
 ## Opening the project
 
+Run
+
+```
+make style-code
+```
+
+To generate the needed generated sources.
```

Comment:
If it's possible to make this comment a bit more specific, that would be good, because after reading it I still don't know what make style-code does.